tests/events/server/storage/test_event_persister.py

```diff
@@ -3,6 +3,7 @@
 from typing import AsyncGenerator, Optional, Sequence
 from uuid import UUID, uuid4
 
+import pendulum
```

Comment:
please use `from prefect.types import DateTime` (you can use `.now` on that the same way) per https://github.com/PrefectHQ/prefect/issues/16910